.github/workflows/deno-ci.yml

```diff
@@ -0,0 +1,17 @@
+name: Deno
+
+on: [push]
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: install Deno
+        run: curl -fsSL https://deno.land/x/install/install.sh | sh
+        env:
+          DENO_VERSION: 0.41.0
```

Comment:
https://github.com/denolib/setup-deno

Reply:
Nice, thank you for the suggestion :tada: